start.sh

```diff
@@ -1,6 +1,9 @@
 KALATORI_HOST="127.0.0.1:16726" \
```

Comment:
```suggestion
KALATORI_CONFIG="configs/chopsticks.toml" \
KALATORI_HOST="127.0.0.1:16726" \
```

Reply:
The funny thing is, without chopsticks setup it just runs. With chopsticks it requires all that messy setup that won't even be in prod. Maybe we want several start scripts here?